torch/utils/data/datapipes/_hook_iterator.py

```diff
@@ -161,7 +168,7 @@ def wrap_generator(*args, **kwargs):
                         _check_iterator_valid(datapipe, iterator_id)
                         response = gen.send(request)
             except StopIteration as e:
-                return e.value
+                return e.value  # noqa: B901
```

Comment:
Do we need to return anything from here?

Reply:
actually nope